files/default/neutron-ha-tool.py

```diff
@@ -233,6 +236,10 @@ def run(args):
     else:
         raise ValueError('Invalid agent_selection_mode')
 
+    if args.target_agent is not None:
+        Configuration.agent_picker_class = SingleAgentPicker
+        SingleAgentPicker.agent_selection_value = args.target_agent
```

Comment:
Agreed - this is related to [what I wrote 2 days ago](https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/9#discussion_r94273578), before I noticed these older comments.